### ISSUE TITLE
rake kontena:reset_admin throws error

### ISSUE THREAD

**Issue by jakolehm** [kontena/kontena#2168]
```
/app $ rake kontena:reset_admin
I, [2017-04-18T09:31:50.629857 #159]  INFO -- : Celluloid 0.17.3 is running in BACKPORTED mode. [ http://git.io/vJf3J ]
rake aborted!
NameError: uninitialized constant Celluloid::Internals::Logger::INFO
Did you mean?  TZInfo
/app/app/services/watchdog.rb:14:in `<class:Watchdog>'
/app/app/services/watchdog.rb:1:in `<top (required)>'
/app/app/boot.rb:24:in `block in require_glob'
/app/app/boot.rb:23:in `each'
/app/app/boot.rb:23:in `require_glob'
/app/app/boot.rb:34:in `<top (required)>'
lib/tasks/reset_admin.rake:8:in `require_relative'
lib/tasks/reset_admin.rake:8:in `block (2 levels) in <top (required)>'
Tasks: TOP => kontena:reset_admin
(See full trace by running task with --trace)
```

**Comment by SpComb** [comment]
This happens because something in the `server/lib/tasks/reset_admin.rake` is requiring `celluloid` before the `app/boot.rb` `require 'celluloid/current'` is hit, which means that celluloid runs in the backported mode, and exposes some internal `Logger` class/module in each Actor class's namespace. That doesn't get exposed in the non-backported mode with the requires in the right order, and the `Logger` in the watchdog actor's class refers to the stdlib logger...

**Comment by kke** [comment]
There's `Bundler.require` which loads everything in Gemfile that hasn't been set `require: false`.

**Comment by SpComb** [comment]
There was also a `require 'celluloid'` in the `lib/tasks/environment.rake`.

**Comment by SpComb** [comment]
Fixed by #2169 but there might also be other things running the wrong celluloid variant if they use `Bundler.require`, such as the `kontena-console`.
